src/components/ui/date-range-picker/date-range-picker.tsx

```diff
@@ -46,6 +48,7 @@ export const DateRangePicker: React.FC<DateRangePickerProps> = ({
         maxDate={maxDate}
         containerClassName={containerClassName}
       />
+      {error != null && <p className='text-red-500 text-sm'>{error.start_date}</p>}
```

Comment:
must also show end date error?

Reply:
The reason I display only the start_date is because the error message returned from both start_date and end_date is identical.